projects/observability/src/pages/explorer/explorer.component.test.ts

```diff
@@ -115,11 +115,17 @@ describe('Explorer component', () => {
     spectator.tick();
     patchRouterNavigateForTest(spectator);
     detectQueryChange();
-    querySpy = spectator.inject(GraphQlRequestService).queryImmediately;
+    querySpy = spectator.inject(GraphQlRequestService).query;
   };
 
   test('fires query on init for traces', fakeAsync(() => {
-    init();
+    init({
+      providers: [
```

Comment:
If we're already doing on 75 why do we need to override in each test?

Reply:
I have seen sometimes they keep record of calls made by other tests. Providing in the test fixes the issue. It is weird and not sure if i am missing something.

Reply:
hmm - will try to pull and see if I can figure out why that is, because we shouldn't need to do this. don't hold for this though, since I might forget...